### PR TITLE
Store content based on the unicode support flag

### PR DESCRIPTION
**Pull request by darshanasbg**
### Proposed changes in this pull request

$subject. This handles the core logic for the config introduced in:
- https://github.com/wso2/carbon-identity-framework/pull/6142

This temporal behavior enables onboarding the change,
* https://github.com/wso2/carbon-identity-framework/pull/6120
* https://github.com/wso2-extensions/identity-event-handler-notification/pull/280
in a graceful manner when the previous schema is already in use.

Related to,
* https://github.com/wso2/product-is/issues/21235

### When should this PR be merged

After merging https://github.com/wso2/carbon-identity-framework/pull/6142

### Follow up actions

N/A

### CODE REVIEW

File: components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/util/I18nEmailUtil.java [hunk 1]
```diff
@@ -281,4 +285,31 @@ public static String getNotificationLocale() {
                 ? IdentityUtil.getProperty(I18nMgtConstants.NOTIFICATION_DEFAULT_LOCALE)
                 : I18nMgtConstants.DEFAULT_NOTIFICATION_LOCALE;
     }
+
+    public static byte[] getContentByteArray(NotificationTemplate notificationTemplate) {
```
Comment (UdeshAthukorala) [inline components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/util/I18nEmailUtil.java:289]: Shall we add the method comments

File: components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/util/I18nEmailUtil.java [hunk 2]
```diff
@@ -281,4 +285,31 @@ public static String getNotificationLocale() {
...
+
+        String[] templateContent = new String[]{notificationTemplate.getSubject(), notificationTemplate.getBody(),
+                notificationTemplate.getFooter()};
+        String content = new Gson().toJson(templateContent);
+        return content.getBytes(StandardCharsets.UTF_8);
+    }
+
+    public static void setContent(InputStream contentStream, NotificationTemplate notificationTemplateResult) throws
```
Comment (UdeshAthukorala) [inline components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/util/I18nEmailUtil.java:297]: Shall we add the method comment

**Review by UdeshAthukorala — DISMISSED** [review]

File: components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/util/I18nEmailUtil.java (hunk 2 above)
Comment (darshanasbg) [reply components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/util/I18nEmailUtil.java]: Fixed with: https://github.com/wso2-extensions/identity-event-handler-notification/pull/280/commits/d39163fecfa8daae90fe3e3ebe3b70b805b84349

File: components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/util/I18nEmailUtil.java (hunk 1 above)
Comment (darshanasbg) [reply components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/util/I18nEmailUtil.java]: Fixed with: https://github.com/wso2-extensions/identity-event-handler-notification/pull/280/commits/d39163fecfa8daae90fe3e3ebe3b70b805b84349

**Review by UdeshAthukorala — APPROVED** [review]

**Pull request merged**